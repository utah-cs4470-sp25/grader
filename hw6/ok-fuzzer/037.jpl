show rgba {(if ((! (true || true)) || (if (! false) then false else false)) then (17.93 / 5.25) else rgba {(- (- 12.83)), [61.41][(- 776)], (if (! false) then 2.41 else 61.85), (- (- 50.06))}.g), (- (if (! (false != false)) then (- [61.2][431]) else (if (56.57 == 37.49) then (50.53 / 50.63) else (if (773 == 484) then 95.87 else (- 2.15))))), ((98.69 - (- 7.9)) * [rgba {41.17, 3.56, 52.38, 73.83}.g, 8.78][(322 % 723)]), ((- (- (- 97.7))) * (26.12 * (- 89.61)))}

struct a {
a : float
b : rgba[,]
}
struct b {
a : int
}
struct c {
a : bool
b : a
}
show [(((! (436 < 428)) == (59.54 < (29.53 / (if false then 100.0 else 1.77)))) != true), (if true then false else (false && (! (508 > 478)))), ((((- 15.99) != (41.78 + 61.52)) && (! (46.08 < (if true then 71.92 else 49.67)))) == (! (false && (if true then false else true))))]

struct d {
a : void
b : c
}
show (450 >= (if (((57.83 / 70.41) % 69.82) <= (- (if (47 == 779) then 92.48 else 45.86))) then [[105, 141][(- 276)], 620][(- (340 * 699))] else b {(if (420 == 439) then 302 else (- 437))}.a))

show b {(if (21.23 == 34.17) then (- b {101}.a) else 589)}.a

show true

show (if (! true) then [54.51, (if (! true) then (60.96 * 81.64) else 40.5), rgba {14.11, 62.73, 3.1, (- (- 69.94))}.a] else [98.69, (66.72 / 98.76)])

struct e {
a : d
b : void
}
struct f {
a : rgba
}
show 75.29

show (- ((if (((- 851) >= (- 414)) == (! true)) then ((- (- 46.58)) - (55.52 + 69.05)) else (- (- 90.65))) % 44.81))

show b {(if false then (if true then (334 - 527) else 180) else (if (true != false) then 380 else 841))}.a

struct g {
a : c
}
struct h {
a : float
b : int
}
show [(h {32.92, (115 % 23)}.b != (- ([449, 970][(421 * 860)] % (- (- 495)))))]

