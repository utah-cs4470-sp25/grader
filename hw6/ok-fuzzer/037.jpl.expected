(ShowCmd (StructLiteralExpr (StructType rgba) rgba (IfExpr (FloatType) (BinopExpr (BoolType) (UnopExpr (BoolType) ! (BinopExpr (BoolType) (TrueExpr (BoolType)) || (TrueExpr (BoolType)))) || (IfExpr (BoolType) (UnopExpr (BoolType) ! (FalseExpr (BoolType))) (FalseExpr (BoolType)) (FalseExpr (BoolType)))) (BinopExpr (FloatType) (FloatExpr (FloatType) 17) / (FloatExpr (FloatType) 5)) (DotExpr (FloatType) (StructLiteralExpr (StructType rgba) rgba (UnopExpr (FloatType) - (UnopExpr (FloatType) - (FloatExpr (FloatType) 12))) (ArrayIndexExpr (FloatType) (ArrayLiteralExpr (ArrayType (FloatType) 1) (FloatExpr (FloatType) 61)) (UnopExpr (IntType) - (IntExpr (IntType) 776))) (IfExpr (FloatType) (UnopExpr (BoolType) ! (FalseExpr (BoolType))) (FloatExpr (FloatType) 2) (FloatExpr (FloatType) 61)) (UnopExpr (FloatType) - (UnopExpr (FloatType) - (FloatExpr (FloatType) 50)))) g)) (UnopExpr (FloatType) - (IfExpr (FloatType) (UnopExpr (BoolType) ! (BinopExpr (BoolType) (FalseExpr (BoolType)) != (FalseExpr (BoolType)))) (UnopExpr (FloatType) - (ArrayIndexExpr (FloatType) (ArrayLiteralExpr (ArrayType (FloatType) 1) (FloatExpr (FloatType) 61)) (IntExpr (IntType) 431))) (IfExpr (FloatType) (BinopExpr (BoolType) (FloatExpr (FloatType) 56) == (FloatExpr (FloatType) 37)) (BinopExpr (FloatType) (FloatExpr (FloatType) 50) / (FloatExpr (FloatType) 50)) (IfExpr (FloatType) (BinopExpr (BoolType) (IntExpr (IntType) 773) == (IntExpr (IntType) 484)) (FloatExpr (FloatType) 95) (UnopExpr (FloatType) - (FloatExpr (FloatType) 2)))))) (BinopExpr (FloatType) (BinopExpr (FloatType) (FloatExpr (FloatType) 98) - (UnopExpr (FloatType) - (FloatExpr (FloatType) 7))) * (ArrayIndexExpr (FloatType) (ArrayLiteralExpr (ArrayType (FloatType) 1) (DotExpr (FloatType) (StructLiteralExpr (StructType rgba) rgba (FloatExpr (FloatType) 41) (FloatExpr (FloatType) 3) (FloatExpr (FloatType) 52) (FloatExpr (FloatType) 73)) g) (FloatExpr (FloatType) 8)) (BinopExpr (IntType) (IntExpr (IntType) 322) % (IntExpr (IntType) 723)))) (BinopExpr (FloatType) (UnopExpr (FloatType) - (UnopExpr (FloatType) - (UnopExpr (FloatType) - (FloatExpr (FloatType) 97)))) * (BinopExpr (FloatType) (FloatExpr (FloatType) 26) * (UnopExpr (FloatType) - (FloatExpr (FloatType) 89))))))
(StructCmd a a (FloatType) b (ArrayType (StructType rgba) 2))
(StructCmd b a (IntType))
(StructCmd c a (BoolType) b (StructType a))
(ShowCmd (ArrayLiteralExpr (ArrayType (BoolType) 1) (BinopExpr (BoolType) (BinopExpr (BoolType) (UnopExpr (BoolType) ! (BinopExpr (BoolType) (IntExpr (IntType) 436) < (IntExpr (IntType) 428))) == (BinopExpr (BoolType) (FloatExpr (FloatType) 59) < (BinopExpr (FloatType) (FloatExpr (FloatType) 29) / (IfExpr (FloatType) (FalseExpr (BoolType)) (FloatExpr (FloatType) 100) (FloatExpr (FloatType) 1))))) != (TrueExpr (BoolType))) (IfExpr (BoolType) (TrueExpr (BoolType)) (FalseExpr (BoolType)) (BinopExpr (BoolType) (FalseExpr (BoolType)) && (UnopExpr (BoolType) ! (BinopExpr (BoolType) (IntExpr (IntType) 508) > (IntExpr (IntType) 478))))) (BinopExpr (BoolType) (BinopExpr (BoolType) (BinopExpr (BoolType) (UnopExpr (FloatType) - (FloatExpr (FloatType) 15)) != (BinopExpr (FloatType) (FloatExpr (FloatType) 41) + (FloatExpr (FloatType) 61))) && (UnopExpr (BoolType) ! (BinopExpr (BoolType) (FloatExpr (FloatType) 46) < (IfExpr (FloatType) (TrueExpr (BoolType)) (FloatExpr (FloatType) 71) (FloatExpr (FloatType) 49))))) == (UnopExpr (BoolType) ! (BinopExpr (BoolType) (FalseExpr (BoolType)) && (IfExpr (BoolType) (TrueExpr (BoolType)) (FalseExpr (BoolType)) (TrueExpr (BoolType))))))))
(StructCmd d a (VoidType) b (StructType c))
(ShowCmd (BinopExpr (BoolType) (IntExpr (IntType) 450) >= (IfExpr (IntType) (BinopExpr (BoolType) (BinopExpr (FloatType) (BinopExpr (FloatType) (FloatExpr (FloatType) 57) / (FloatExpr (FloatType) 70)) % (FloatExpr (FloatType) 69)) <= (UnopExpr (FloatType) - (IfExpr (FloatType) (BinopExpr (BoolType) (IntExpr (IntType) 47) == (IntExpr (IntType) 779)) (FloatExpr (FloatType) 92) (FloatExpr (FloatType) 45)))) (ArrayIndexExpr (IntType) (ArrayLiteralExpr (ArrayType (IntType) 1) (ArrayIndexExpr (IntType) (ArrayLiteralExpr (ArrayType (IntType) 1) (IntExpr (IntType) 105) (IntExpr (IntType) 141)) (UnopExpr (IntType) - (IntExpr (IntType) 276))) (IntExpr (IntType) 620)) (UnopExpr (IntType) - (BinopExpr (IntType) (IntExpr (IntType) 340) * (IntExpr (IntType) 699)))) (DotExpr (IntType) (StructLiteralExpr (StructType b) b (IfExpr (IntType) (BinopExpr (BoolType) (IntExpr (IntType) 420) == (IntExpr (IntType) 439)) (IntExpr (IntType) 302) (UnopExpr (IntType) - (IntExpr (IntType) 437)))) a))))
(ShowCmd (DotExpr (IntType) (StructLiteralExpr (StructType b) b (IfExpr (IntType) (BinopExpr (BoolType) (FloatExpr (FloatType) 21) == (FloatExpr (FloatType) 34)) (UnopExpr (IntType) - (DotExpr (IntType) (StructLiteralExpr (StructType b) b (IntExpr (IntType) 101)) a)) (IntExpr (IntType) 589))) a))
(ShowCmd (TrueExpr (BoolType)))
(ShowCmd (IfExpr (ArrayType (FloatType) 1) (UnopExpr (BoolType) ! (TrueExpr (BoolType))) (ArrayLiteralExpr (ArrayType (FloatType) 1) (FloatExpr (FloatType) 54) (IfExpr (FloatType) (UnopExpr (BoolType) ! (TrueExpr (BoolType))) (BinopExpr (FloatType) (FloatExpr (FloatType) 60) * (FloatExpr (FloatType) 81)) (FloatExpr (FloatType) 40)) (DotExpr (FloatType) (StructLiteralExpr (StructType rgba) rgba (FloatExpr (FloatType) 14) (FloatExpr (FloatType) 62) (FloatExpr (FloatType) 3) (UnopExpr (FloatType) - (UnopExpr (FloatType) - (FloatExpr (FloatType) 69)))) a)) (ArrayLiteralExpr (ArrayType (FloatType) 1) (FloatExpr (FloatType) 98) (BinopExpr (FloatType) (FloatExpr (FloatType) 66) / (FloatExpr (FloatType) 98)))))
(StructCmd e a (StructType d) b (VoidType))
(StructCmd f a (StructType rgba))
(ShowCmd (FloatExpr (FloatType) 75))
(ShowCmd (UnopExpr (FloatType) - (BinopExpr (FloatType) (IfExpr (FloatType) (BinopExpr (BoolType) (BinopExpr (BoolType) (UnopExpr (IntType) - (IntExpr (IntType) 851)) >= (UnopExpr (IntType) - (IntExpr (IntType) 414))) == (UnopExpr (BoolType) ! (TrueExpr (BoolType)))) (BinopExpr (FloatType) (UnopExpr (FloatType) - (UnopExpr (FloatType) - (FloatExpr (FloatType) 46))) - (BinopExpr (FloatType) (FloatExpr (FloatType) 55) + (FloatExpr (FloatType) 69))) (UnopExpr (FloatType) - (UnopExpr (FloatType) - (FloatExpr (FloatType) 90)))) % (FloatExpr (FloatType) 44))))
(ShowCmd (DotExpr (IntType) (StructLiteralExpr (StructType b) b (IfExpr (IntType) (FalseExpr (BoolType)) (IfExpr (IntType) (TrueExpr (BoolType)) (BinopExpr (IntType) (IntExpr (IntType) 334) - (IntExpr (IntType) 527)) (IntExpr (IntType) 180)) (IfExpr (IntType) (BinopExpr (BoolType) (TrueExpr (BoolType)) != (FalseExpr (BoolType))) (IntExpr (IntType) 380) (IntExpr (IntType) 841)))) a))
(StructCmd g a (StructType c))
(StructCmd h a (FloatType) b (IntType))
(ShowCmd (ArrayLiteralExpr (ArrayType (BoolType) 1) (BinopExpr (BoolType) (DotExpr (IntType) (StructLiteralExpr (StructType h) h (FloatExpr (FloatType) 32) (BinopExpr (IntType) (IntExpr (IntType) 115) % (IntExpr (IntType) 23))) b) != (UnopExpr (IntType) - (BinopExpr (IntType) (ArrayIndexExpr (IntType) (ArrayLiteralExpr (ArrayType (IntType) 1) (IntExpr (IntType) 449) (IntExpr (IntType) 970)) (BinopExpr (IntType) (IntExpr (IntType) 421) * (IntExpr (IntType) 860))) % (UnopExpr (IntType) - (UnopExpr (IntType) - (IntExpr (IntType) 495))))))))
